WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -282,6 +276,64 @@ void StopCoinJoinCommand(StopCoinJoinCommand stopCommand)
 		await WaitAndHandleResultOfTasksAsync(nameof(trackedAutoStarts), trackedAutoStarts.Values.Select(x => x.Task).ToArray()).ConfigureAwait(false);
 	}
 
+	private async Task<IEnumerable<SmartCoin>> SelectCandidateCoinsAsync(IWallet walletToStart, int bestHeight)
```

Comment:
Nit (feel free to ignore):

Previously this method returned a new `CoinsView`. 
Do we want to lose that? 

We can easily wrap the `coinCandidates` in the return. `return new CoinsView(coinCandidates)`
Then we can change the return value of the method.

Reply:
> Do we want to lose that?

I don't know. Why was that? What is the benefit?

Reply:
> What is the benefit?

Now that I looked at the code deeper, I think there are no benefits for my change. Ignore it.